### PR TITLE
Unify some code in the native picker dialog implementation

### PR DESCRIPTION
**Pull request by ethanshar**
## Description
Unify the code that renders the inner WheelPicker inside the the dialog of the Picker (when passing `useNativePicker`). 
We have a different implementation of the dialog per each platform but the content should be the same. 

This PR is a preparation for JIRA WOAUILIB-1843

## Changelog
Unify internal code of native picker (wheel picker)

### CODE REVIEW

File: src/components/picker/PickerDialog.android.js [hunk 1]
```diff
@@ -68,17 +68,6 @@ class PickerDialog extends BaseComponent {
     );
   }
 
-  renderPicker() {
-    const {children, onValueChange, selectedValue, renderNativePicker, wheelPickerProps, testID} = this.props;
-    if (_.isFunction(renderNativePicker)) {
-      return renderNativePicker(this.props);
-    }
-    return (
-      <WheelPicker selectedValue={selectedValue} onChange={onValueChange} {...wheelPickerProps} testID={`${testID}.wheelPicker`}>
```
Comment (lidord-wix) [inline src/components/picker/PickerDialog.android.js:77]: You don't need the `selectedValue`, `onValueChange`, and `wheelPickerProps` in the propTypes anymore :)

File: src/components/picker/PickerDialog.ios.js [hunk 2]
```diff
@@ -39,17 +38,6 @@ class PickerDialog extends BaseComponent {
     );
   }
 
-  renderPicker() {
-    const {children, onValueChange, selectedValue, renderNativePicker, pickerStyle, testID} = this.props;
-    if (_.isFunction(renderNativePicker)) {
-      return renderNativePicker(this.props);
-    }
-    return (
-      <WheelPicker style={pickerStyle} selectedValue={selectedValue} onChange={onValueChange} testID={`${testID}.wheelPicker`}>
```
Comment (lidord-wix) [inline src/components/picker/PickerDialog.ios.js:48]: Same here, you don't need the `selectedValue` and `onValueChange` props

**Review by lidord-wix — COMMENTED** [review]
Cool!
I left small comment :)

File: src/components/picker/PickerDialog.android.js (hunk 1 above)
Comment (ethanshar) [reply src/components/picker/PickerDialog.android.js:77]: Thanks, fixed

File: src/components/picker/PickerDialog.ios.js (hunk 2 above)
Comment (ethanshar) [reply src/components/picker/PickerDialog.ios.js:48]: fixed

**Review by lidord-wix — APPROVED** [review]

**Pull request merged**